downstream/assemblies/hub/assembly-hub-create-api-token.adoc

```diff
@@ -8,10 +8,10 @@ ifdef::context[:parent-context: {context}]
 [role="_abstract"]
 Before you can interact with automation hub by uploading or downloading collections, you must create an API token. The automation hub API token authenticates your `ansible-galaxy` client to the Red Hat automation hub server.
 
-You can create an API token using automation hub *Token management* in automation hub or *API Token Management* in private automation hub (PAH).
+You can create an API token using automation hub *Offline token* management in automation hub or *API token* management in private automation hub.
```

Comment:
We should be using the attribute for private automation hub right? {PrivateHubName}

Reply:
Fixed.